### DISCUSSION TITLE
Border in High Contrast (Light) Theme cut off

### DISCUSSION THREAD

**Discussion by CronJorian** [community/community#64487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="51" alt="image" src="https://github.com/community/community/assets/44020029/f0aade75-380d-4ebf-83bf-94012b88ba28">

As it can be seen, the left side of the profile picture border is cut off in the feed view.
This should be easy to fix and I didn't see anybody mention it yet.

Hope this is the correct place to report it.

**Comment by arafajs** [comment]
Try to Use a smaller profile picture. This will reduce the amount of cropping that is needed, and may prevent the border from being cut off.

**Reply by CronJorian** [reply]
This is nothing I can change, as these are the profile pictures of people I follow.
These are shown in the feed and due to the border overflowing the bounding box of the div, it is cut off.
